src/platforms/javascript/common/session-replay/privacy.mdx

```diff
@@ -15,34 +15,45 @@ There are several ways to deal with personally identifiable information (PII). B
+| ---------------- | ------------------------ | -----------------------------------     | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| block            | string[]                 | `.sentry-block, [data-sentry-block]`    | Redact all elements that match the DOM selector(s). See [Blocking](#blocking) above for an example.                                                                                                 |
+| blockAllMedia    | boolean                  | `true`                                  | Block _all_ media elements (`img`, `svg`, `video`, `object`, `picture`, `embed`, `map`, `audio`).                                                                                                   |
+| ignore           | string[]                 | `.sentry-ignore, [data-sentry-ignore]`  | Ignores all events on the matching input fields. See [Ignoring](#ignoring) above for an example.                                                                                                    |
+| mask             | string[]                 | `.sentry-mask, [data-sentry-mask]`      | Mask all elements that match the given DOM selectors. See [Masking](#masking) section for an example.                                                                                               |
+| maskAllText      | boolean                  | `true`                                  | Mask _all_ text content. Will pass text content through `maskTextFn` before sending to server.                                                                                                      |
+| maskAllInputs    | boolean                  | `true`                                  | Mask values of `<input>` elements. Passes input values through `maskTextFn` before sending to server.                                                                                               |
+| maskTextFn       | (text: string) => string | `(text) => '*'.repeat(text.length)`     | Function to customize how text content is masked before sending to server. By default, masks text with `*`.                                                                                         |
+| unblock          | string[]                 | `.sentry-unblock, [data-sentry-unblock]`| Don't redact any elements that match the DOM selectors. Used to unblock specific media elements that are blocked with `blockAllMedia`.                                                     |
+| unmask           | string[]                 | `.sentry-unmask, [data-sentry-unmask]`  | Unmask all elements that match the given DOM selectors. Used to unmask specific elements that are masked with `maskAllText`.                                                                |
+
+### Deprecated Options
```

Comment:
Just thinking - do we still want to show the deprecated options in the docs? What about putting them in the migration docs and linking there (or similar)?